src/app/session/session-interceptor.service.ts

```diff
@@ -40,12 +40,12 @@ export class SessionInterceptorService implements HttpInterceptor {
 
     return next.handle(req).pipe(
       tap(
-        () => {},
-        response => {
+        () => { console.log(); },
```

Comment:
empty callbacks are not allowed? I guess `null` would also work

Reply:
With null you have an 'error' too "expected and assignment or function"

Reply:
hmm allright. Try adding `() => { /**/ }`